### DISCUSSION TITLE
Teleport to the Central Island of Resource World's End Dimension

### DISCUSSION THREAD

**Discussion by g-g-glitch** [Stardust-Labs-MC/StardustMC#48] [Ideas]
I think it would be beneficial for players to be able to directly teleport to the central island of Resource World's End dimension so that they can respawn the Ender Dragon without damaging the structures built on the central island in the main End dimension.

**Comment by aisukuma** [comment]
Done. By doing this, you get teleported on top the bedrock portal to the overworld in the resource end.
